packages/core/src/repository/query/network.query.ts

```diff
@@ -0,0 +1,93 @@
+import { ParameterType } from '../../data';
+import {KeyQuery} from "./query";
+
```

Comment:
**Nit:** What do you think about adding a type alias for `Record<string, ParameterType>`?

```suggestion

export type NetworkParams = Record<string, ParameterType>;

```